### DISCUSSION TITLE
Is this production ready, and if so, how much support can we expect?

### DISCUSSION THREAD

**Discussion by dharmaturtle** [microsoft/coyote#393] [Q&A] [answered]
Last thing people wanna do is switch over all their `Tasks`, only to find 1 year later that Microsoft Research is trying something new, different, and incompatible with prior products :D

If the answers are "yes" and "hella good support", it should be declared as such in the Readme.

**Comment by akashlal** [comment, marked as answer]
Coyote is production ready. It is based off the [PSharp](https://github.com/p-org/PSharp) project that is being used by several teams in Azure. Some teams have already started pulling the latest Coyote bits, and others are expected to "upgrade" to Coyote soon.
 
 

In terms of support, see our [FAQs](https://microsoft.github.io/coyote/learn/overview/faq#how-is-coyote-being-maintained-and-what-level-of-support-can-we-expect). Coyote is an open source project. Microsoft is both a [user](https://microsoft.github.io/coyote/case-studies/azure-batch-service) of Coyote and also continues to invest in it.

**Comment by pdeligia** [comment]
> Last thing people wanna do is switch over all their `Tasks`, only to find 1 year later that Microsoft Research is trying something new, different, and incompatible with prior products :D
> 
> If the answers are "yes" and "hella good support", it should be declared as such in the Readme.

Old thread, but for the benefit of any future reader, Coyote is not asking you to replace the task type anymore, binary rewriting (during testing only) automatically does this for you so your production code can stay unchanged.
